io/MalReader.io

```diff
@@ -28,7 +28,7 @@ MalReader := Object clone do (
     numberRegex := Regex with("^-?[0-9]+$")
 
     read_string := method(token,
-        token exSlice(1, -1) replaceSeq("\\\"", "\"") replaceSeq("\\n", "\n") replaceSeq("\\\\", "\\")
+        token exSlice(1, -1) replaceSeq("\\\\", "\u029e") replaceSeq("\\\"", "\"") replaceSeq("\\n", "\n") replaceSeq("\u029e", "\\")
```

Comment:
Io doesn't support the `\uNNNN` escape sequence:

```
# io
Io 20110905
Io> ("u029e" == "\u029e")
==> true
Io> "\u029e" size
==> 5
```

Reply:
Chances are you'll have to cook up something similar to https://github.com/kanaka/mal/blob/e27439c7244c32d928000fa1daaeb7e87d4a34b7/gst/util.st#L14

Reply:
Oh right, what was I thinking! I have pushed a fix. Thanks @wasamasa for the help